adapter/src/main/resources/Dockerfile

```diff
@@ -41,14 +41,16 @@ ARG MOTD="\n\
 RUN \
     addgroup -S -g ${MG_USER_GROUP_ID} ${MG_USER_GROUP} \
     && adduser -S -u ${MG_USER_ID} -h ${MG_USER_HOME} -G ${MG_USER_GROUP} ${MG_USER} \
-    && mkdir ${MG_USER_HOME}/logs && chown -R ${MG_USER}:${MG_USER_GROUP} ${MG_USER_HOME}/logs  \
+    && mkdir ${MG_USER_HOME}/logs && chown -R ${MG_USER}:${MG_USER_GROUP} ${MG_USER_HOME}/logs \
+    && mkdir -p ${MG_USER_HOME}/artifacts/apis && chown -R ${MG_USER}:${MG_USER_GROUP} ${MG_USER_HOME}/artifacts \
```

Comment:
chown -R ${MG_USER}:${MG_USER_GROUP} Already granted in line 44?

Reply:
In line 44 chown is executed for `${MG_USER_HOME}/logs` directory. In line 45 I have added chown for `${MG_USER_HOME}/artifacts` directory

Reply:
Shall we do it the same way, that we have done in the Enforcer docker file? mkdir all paths and later chown MG_USER_HOME. Having the same way is better in each Dockerfiless.

Reply:
fixed, changed all Dockerfiles to `mkdir` the required paths first and then `chown` the `MG_USER_HOME `